src/oscar/apps/dashboard/vouchers/views.py

```diff
@@ -171,7 +171,7 @@ def get_form_kwargs(self):
 
     def get_initial(self):
         voucher = self.get_voucher()
-        offer = voucher.offers.all()[0]
+        offer = voucher.offers.first()
```

Comment:
When no offers exist this now causes an AttrbuteError which used to be an IndexError. Please confirm this is harmless.

Reply:
The dashboard `VoucherUpdateView` assumes there is always one offer associated with a voucher (which is the only way you can create one via the dashboard). The behaviour is unchanged in that both the old `IndexError` and the new `AttributeError` would be unhandled if you try to edit a voucher whose offer has somehow been deleted. TLDR: yeah it's fine.